src/api/app/controllers/search_controller.rb

```diff
@@ -269,16 +269,16 @@ def find_attribute(namespace, name)
 
     # gather the relation for attributes depending on project/package combination
     if params[:package]
-      if params[:project]
-        attribs = Package.get_by_project_and_name(params[:project], params[:package]).attribs
+      attribs = if params[:project]
```

Comment:
strange indentation, wouldn't it be better to have `if`, `elseif` and `else` aligned?